clippy_dev/src/main.rs

```diff
@@ -51,6 +56,26 @@ fn main() {
                         .help("Checks that util/dev update_lints has been run. Used on CI."),
                 ),
         )
+        .subcommand(
+            SubCommand::with_name("new_lint")
+                .about("Create new lint and run util/dev update_lints")
+                .arg(
+                    Arg::with_name("type")
```

Comment:
What are your thoughts on calling this argument `pass`?

Reply:
I think that's reasonable, I can do that if there are no objections